projects/github-actions/.gitignore

```diff
@@ -0,0 +1,2 @@
+composer.lock
+vendor/
```

Comment:
I wonder whether we actually should have type-level .gitignores, particularly ones we'd likely put on almost every type.

We probably want `vendor/` ignored everywhere, even plugins like Jetpack will have it coupled with appropriate .gitattributes to include certain packages in the mirror.

`composer.lock` we'll probably want ignored on everything except plugins.

And we could say the same for `node_modules/` and `yarn.lock`, really.

Reply:
We can do that in a follow-up. Sounds good.

Reply:
[`3e43301` (#18504)](https://github.com/Automattic/jetpack/pull/18504/commits/3e43301fcb28dca861b9b569b4d03d06163a543e) to move `vendor` upstream.

Reply:
https://github.com/Automattic/jetpack/issues/18712 for tracking